src/components/PeopleTable.tsx

```diff
@@ -1,645 +1,88 @@
-          <td>m</td>
-          <td>1832</td>
-          <td>1905</td>
-          <td>
-            <a
-              className="has-text-danger"
-              href="#/people/maria-van-brussel-1801"
-            >
-              Maria van Brussel
-            </a>
-          </td>
-          <td>
-            <a href="#/people/carel-haverbeke-1796">Carel Haverbeke</a>
-          </td>
-        </tr>
+    const newSearchParams = new URLSearchParams(searchParams);
 
-        <tr data-cy="person">
-          <td>
-            <a className="has-text-danger" href="#/people/maria-sturm-1835">
-              Maria Sturm
-            </a>
-          </td>
-          <td>f</td>
-          <td>1835</td>
-          <td>1917</td>
-          <td>Seraphina Spelier</td>
-          <td>Charles Sturm</td>
-        </tr>
+    if (!sort && sort !== sortColumn) {
+      newSearchParams.set('sort', sortColumn);
+    } else if (sort === sortColumn && order !== 'desc') {
+      newSearchParams.set('order', 'desc');
+    } else {
+      newSearchParams.delete('order');
+      newSearchParams.delete('sort');
+    }
 
-        <tr data-cy="person">
-          <td>
-            <a
-              className="has-text-danger"
-              href="#/people/emma-de-milliano-1876"
-            >
-              Emma de Milliano
-            </a>
-          </td>
-          <td>f</td>
-          <td>1876</td>
-          <td>1956</td>
-          <td>Sophia van Damme</td>
-          <td>Petrus de Milliano</td>
-        </tr>
+    setSearchParams(newSearchParams);
+  };
 
-        <tr data-cy="person">
-          <td>
-            <a href="#/people/emile-haverbeke-1877">Emile Haverbeke</a>
-          </td>
-          <td>m</td>
-          <td>1877</td>
-          <td>1968</td>
-          <td>
-            <a className="has-text-danger" href="#/people/maria-sturm-1835">
-              Maria Sturm
-            </a>
-          </td>
-          <td>
-            <a href="#/people/carolus-haverbeke-1832">Carolus Haverbeke</a>
-          </td>
-        </tr>
-      </tbody>
-    </table>
+  return (
+    <>
+      {loading ? (
+        <Loader />
+      ) : (
+        <table
+          data-cy="peopleTable"
+          className="table is-striped is-hoverable is-narrow is-fullwidth"
+        >
+          <thead>
+            <tr>
+              {sortColumns.map(column => (
+                <th key={column}>
+                  <span className="is-flex is-flex-wrap-nowrap">
+                    {Column[column].charAt(0).toUpperCase() +
+                      Column[column].slice(1)}
+                    <a href="" onClick={e => handleLinkOnClick(e, column)}>
+                      <span className="icon">
+                        <i
+                          className={cn('fas', {
+                            'fa-sort': sort !== column,
+                            'fa-sort-up': sort === column && order !== 'desc',
+                            'fa-sort-down': sort === column && order,
```

Comment:
Same here, create variable for each conditional

Reply:
good review, thx. Ok, will follow.